pkg/bpfassets/attacher/attacher.go

```diff
@@ -149,20 +148,27 @@ func getLibbpfObjectFilePath() (string, error) {
 		endianness = "bpfeb"
 	}
 	filename := fmt.Sprintf(objectFilename, endianness)
-	bpfassetsPath := fmt.Sprintf("%s/%s", bpfAssesstsLocation, filename)
+	bpfassetsPath := fmt.Sprintf("%s/%s", bpfAssestsLocation, filename)
 	_, err := os.Stat(bpfassetsPath)
 	if err != nil {
-		var absPath string
-		// try relative path
-		absPath, err = filepath.Abs(bpfAssesstsLocalPath)
+		// attempt to find the bpf assets in the same directory as the binary
+		// this is useful for running locally
+		var matches []string
+		err = filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
+			if info.Name() == filename {
+				matches = append(matches, path)
+			}
+			return nil
+		})
 		if err != nil {
-			return "", err
+			return "", fmt.Errorf("failed to find bpf object file: %v", err)
 		}
-		bpfassetsPath = fmt.Sprintf("%s/%s", absPath, filename)
-		_, err = os.Stat(bpfassetsPath)
-		if err != nil {
-			return "", err
+		fmt.Println(matches)
+		if len(matches) < 1 {
+			return "", fmt.Errorf("failed to find bpf object file: no matches found")
 		}
+		klog.Infof("found bpf object file: %s", matches[0])
+		return matches[0], nil
```

Comment:
since anyway we are using the first match, why not return early when `info.Name() == filename` ?